content/en/docs/File Format/Types/VariantEncoding.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
This is really nice to have variant on the webpage now 

<img width="1360" height="1168" alt="Screenshot 2025-12-03 at 3 21 03 PM" src="https://github.com/user-attachments/assets/2f0c91de-f39c-46ed-b46f-1fcb48c5ca49" />